lib/src/default_index/composite.rs

```diff
@@ -410,12 +414,6 @@ impl AsCompositeIndex for CompositeIndex<'_> {
 }
 
 impl Index for CompositeIndex<'_> {
-    /// Suppose the given `commit_id` exists, returns the minimum prefix length
```

Comment:
I removed some comments like this one since the functions are documented in the trait declaration.